app/keepers/keepers.go

```diff
@@ -621,7 +621,7 @@ func NewAppKeeper(
 		app.LeveragelpKeeper,
 		app.StablestakeKeeper,
 	)
-	app.AmmKeeper.SetTierKeeper(&app.TierKeeper)
+	app.AmmKeeper.SetTierKeeper(app.TierKeeper)
```

Comment:
If we set whole value here and if there are changes in tier keeper, will it reflect here ?

Reply:
Yes, I made app.TierKeeper pointer as well